lib/tests/test_gpg.rs

```diff
@@ -0,0 +1,162 @@
+use std::fs::Permissions;
+use std::io::Write;
+#[cfg(unix)]
+use std::os::unix::prelude::PermissionsExt;
+use std::path::PathBuf;
+use std::process::Stdio;
+
+use assert_matches::assert_matches;
+use jj_lib::gpg_signing::GpgBackend;
+use jj_lib::signing::{SigStatus, SignError, SigningBackend, Verification};
+use once_cell::sync::Lazy;
+
+static GPG_HOME: Lazy<PathBuf> = Lazy::new(|| {
+    let dir = tempfile::Builder::new()
+        .prefix("jj-test-")
+        .tempdir()
+        .unwrap()
+        .into_path();
```

Comment:
Maybe you know, but this temporary directory isn't removed at exit.

Reply:
The tempfile crate implements the Drop trait which cleans up the file. This should be fine!

Reply:
It's static object, so `Drop` won't be called afaik.